website/docs/reference/widgets/input.md

```diff
@@ -1,258 +1,118 @@
-
-
-#### Data type
-
-As the name suggests, data type allows the user to set the type of data you want in the input box, like- 
-
-* Single-line text
-* Multi-line text
-* Number
-* Password
-* Email
-
-The input box shows an error "invalid input" if the entered data doesn't match the data type or disable the invalid data type input. (If you select the data type as a number, you won't be able to enter text in the input box.)
-
-<VideoEmbed host="youtube" videoId="cMUPIKK1_LI" title="Data type" caption="Data type"/>
-
-#### Label
-
-The property hosts a group of configurations that you can use to associate a display name and define a placement for the widget. These properties are usually useful when you want to design forms that follow a defined alignment for your form fields and give a professional look to your forms. Below are the properties that you can use:
-
-##### Text
-
-It allows you to set the display name of the input box. For example, if you want the user to enter a name in the input box, you can enter the text as "Name."
-
-:::tip
-You can leave the text empty if you don't want any display name for your input widget.
-:::
-
-##### Position
-
-It allows you to specify the placement of the label. You can select one of the available options:
-
-* Top - It allows you to align the text at the top of the input box.
-* Left - It aligns the text to the left of the input box. When you select **Left** alignment, you get additional settings that you can use to control the alignment and define the text's width.
-  * Alignment - With the help of alignment, you can define the placement of the text in accordance with the position of the input box. You can choose:
-    * Left - It aligns the text to the widget's left boundary that's away from the input box.
-    * Right - It aligns the text closer to the input box.
-  * Width - With the help of width, you can define the **number of columns** in the **grid** that surrounds the widget. You can specify how close or far the text can be placed to the input box.
-* Auto - It automatically adjusts the position of the text based on the input box's height.
-
-:::info
-Columns are the dashed lines (-----) that surround a widget when you try to drag, and drop it on the canvas.
-:::
+## Input validation
 
-<VideoEmbed host="youtube" videoId="7VFebef9JZg" title="How to set label properties?" caption="How to set label properties?"/>
+Input widgets gather user-provided data, which can be accessed on the widget's `text` property. However, to prevent polluting your dataset with messy or incorrectly formatted data, it's important to guide users toward providing input in a usable format.
```

Comment:
I'm inclined to keep a couple of the original bits;
1) `text` -> **Text**
    - In this case, "text" is referring to the reference property `Input1.text` which is accessed with code; usually when we use the **Bold** style, it's referring to a property field in the properties pane that the user puts data into.
2)  In the second part, I like the verb "help" or "guide" because the following subhead is about communicating to the user with tooltips/placeholders to let them know what is expected; some of the validation is about telling them what is right/wrong, but "help" is a bit softer

I definitely like removing that big bit in the middle though; I'll find a middle-ground between these, let me know what you think:

> Input widgets gather user-provided data, which can be accessed using the widget's `text` property. First, however, it's important to help users provide clean and well-formatted data.

Reply:
sounds perfect :)